docs/openapi/components/schemas/common/GAPInspection.yml

```diff
@@ -0,0 +1,427 @@
+$linkedData:
+  term: GAPInspection
+  '@id': https://w3id.org/traceability#GAPInspection
+title: GAP Inspection
+description: A harmonized GAP (Good Agricultural Practices) or GAP Plus+ inspection report.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - GAPInspection
+      - type: string
+        const:
+          - GAPInspection
+  GAPPlus:
+    title: GAP Plus
+    description: True/false statement of whether this inspection is a harmonized GAP Plus+ inspection.
```

Comment:
Following on above, it seems that either `title: GAP Plus` should be `title: GAP Plus+` or `harmonized GAP Plus+ inspection` should be `harmonized GAP Plus inspection`...